apps/coordinator/src/clients/transactions.ts

```diff
@@ -130,12 +141,33 @@ export const usePendingTransactions = () => {
     isLoading,
     error,
     refetch: () => {
-      // Refetch all transaction queries
       transactionQueries.forEach((query) => query.refetch());
     },
   };
 };
 
+// Hook for processed pending transactions - uses selector
+export const usePendingTransactions = () => {
+  const walletAddresses = useSelector(getWalletAddresses);
+  const rawPendingQuery = useRawPendingTransactions();
+
+  const transactions = useMemo(() => {
+    if (!rawPendingQuery.transactions) return [];
+    return selectProcessedTransactions(
+      rawPendingQuery.transactions,
+      walletAddresses,
+      "unconfirmed",
+    );
+  }, [rawPendingQuery.transactions, walletAddresses]);
```

Comment:
see, this is the same type of filtering that you're doing for confirmed [here](https://github.com/caravan-bitcoin/caravan/pull/352/files#diff-a6cca066dd7cd585b6683aa19e351c494fac3dd5526ddfcdfb90f42df606248cR78) but this is in the hook making sure you correctly only return the pending ones. Could you use a similar selector filter for only returning confirmed?